### DISCUSSION TITLE
Cannot access Codespaces

### DISCUSSION THREAD

**Discussion by doorgan** [community/community#3554] [Codespaces] [answered]
Hi all,

Since yesterday morning I am not able to access any of my Codespaces.
Yesterday GitHub took me to a 500 error page with no details. Later it would enter in a loop of "Connecting" -> page reloads -> "Starting codespace" -> page reloads -> "Connecting", and ocassionally failing with an empty 401 response.

Right now when I try to access Codespaces, it gets stuck at "Starting the codespace".

status.github.com doesn't show any incidents, so I'm not sure if this is an availability issue, or there's some issue with my account.

Do you have any idea of what could be causing this?

Best regards.

**Comment by srivatsn** [comment, marked as answer]
There were some issues with Azure yesterday that caused problems for Codespaces. If you are still seeing this today please let us know.

**Reply by doorgan** [reply]
Thanks for your reply. Yes, I'm still not able to access to Codespaces right now. It gets hung here:
![image](https://user-images.githubusercontent.com/6147776/111386467-107aee00-868b-11eb-8eaa-d2e35e2c6e87.png)

**Reply by srivatsn** [reply]
@jkeech @edgonmsft can we take a look?

**Reply by doorgan** [reply]
Thanks for looking into it, it seems to be working again!
I'll reach back if I find any issues, but so far it seems to be fixed.

Thanks!

**Reply by jkeech** [reply]
Glad to hear it's working! Please let us know if you see any more issues.

**Reply by kunurudra41** [reply]
I’m also facing this issue currently, do you have any solutions for this  and it stocked here
![IMG_0148](https://github.com/user-attachments/assets/745dcb44-f068-4991-a7c6-7a3e71651398)

**Reply by Lad-Trushil** [reply]
I still don't see a solution here... What is to be done can you please help?

**Comment by diidhul** [comment]
me too, i cant access codespaces since yesterday, it always kicked me out while setting up the remote connection, it keep shows this error messages 
![Screenshot 2024-01-01 005637](https://github.com/community/community/assets/74520695/e4eb45fe-fc08-4f67-8098-d035c861d6a1)

**Comment by arudaev** [comment]
Reloading the page didn't work. I will comeback tomorrow if the issue persists. It is an interesting issue

![image](https://github.com/community/community/assets/66517969/f1002979-9dfa-40ee-8ef2-2197eee205a5)

**Reply by arudaev** [reply]
The issue has been resolved immediately, by switching the repo from private to public. Whelp, it is what it is 🤷

**Reply by LEKPHET-KUNSUE** [reply]
This seem happened in my old account @Phetdanai

**Comment by hopk8** [comment]
I'm having a similar issue today when trying to access my codespace. The page initially says "setting up codespace" and then reverts to "stopping codespace" and goes no further. I was finally able to make it in, but it's very slow in setting up. When I was in, there was some error and I got disconnected. Not sure if there's something happening on Github's end or mine, but my internet has been working fine today.
<img width="1512" alt="Github Copilot Screenshot 2024-03-18" src="https://github.com/community/community/assets/157246487/b8fbc824-4df1-47ce-8c66-28ab91195840">

**Reply by majicmall** [reply]
I am having the exact issue. How did you resolve it?

**Comment by Vipinbhati28** [comment]
Hello , all 
I'm facing a problem that my codespace is not opening it shows me " Setting Up Your Codespace" .

status.github.com show a fault in codespaces in morning but now doesn't show any fault .

**Comment by MarSou8** [comment]
hi all, I am currently experience same issue here. I cant access to codespace again :( 
<img width="1920" height="782" alt="codespace" src="https://github.com/user-attachments/assets/2a6929c7-1693-42f1-8671-f6384dbf926e" />
